tutorials/Multi_objective_multi_fidelity_BO.ipynb

```diff
@@ -0,0 +1,1416 @@
+    "    model = SingleTaskGP(train_x,\n",
+    "                         train_obj,\n",
+    "                         outcome_transform=Standardize(m=train_obj.shape[-1]))\n",
+    "    mll = ExactMarginalLogLikelihood(model.likelihood, model)\n",
+    "    return mll, model"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Helper function to optimize acquisition function \n",
+    "This is a helper function that initializes, optimizes the acquisition function MOMF and returns the new_x and new_obj. The problem is called from within this helper function.\n",
+    "\n",
+    "A simple initialization heuristic is used to select the 20 restart initial locations from a set of 1024 random points. Multi-start optimization of the acquisition function is performed using LBFGS-B with exact gradients computed via auto-differentiation."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from botorch.utils.multi_objective.box_decompositions.non_dominated import (\n",
+    "    FastNondominatedPartitioning,\n",
+    ")\n",
+    "from botorch.acquisition.multi_objective.monte_carlo import MOMF\n",
+    "from botorch.optim.optimize import optimize_acqf\n",
+    "from botorch.utils.transforms import unnormalize\n",
+    "\n",
+    "\n",
+    "def optimize_MOMF_and_get_obs(\n",
+    "    model,\n",
+    "    train_obj,\n",
+    "    sampler,\n",
+    "    dim_y,\n",
+    "    ref_point,\n",
+    "    standard_bounds,\n",
+    "    BATCH_SIZE,\n",
+    "    cost_call,\n",
+    "):\n",
+    "    \"\"\"Wrapper to call MOMF and optimizes it in a sequential greedy fashion returning a new candidate and evaluation\"\"\"\n",
+    "    partitioning = FastNondominatedPartitioning(\n",
+    "        ref_point=torch.tensor(ref_point).to(**tkwargs), Y=train_obj\n",
+    "    )\n",
+    "    acq_func = MOMF(\n",
+    "        model=model,\n",
+    "        ref_point=ref_point,  # use known reference point\n",
+    "        partitioning=partitioning,\n",
+    "        sampler=sampler,\n",
+    "        cost_call=cost_call,\n",
+    "    )\n",
+    "    # Optimization\n",
+    "    candidates, _ = optimize_acqf(\n",
+    "        acq_function=acq_func,\n",
+    "        bounds=standard_bounds,\n",
+    "        q=BATCH_SIZE,\n",
+    "        num_restarts=20,\n",
+    "        raw_samples=1024,  # used for intialization heuristic\n",
+    "        options={\"batch_limit\": 5, \"maxiter\": 200, \"nonnegative\": True},\n",
+    "        sequential=True,\n",
+    "    )\n",
+    "    # observe new values\n",
+    "    new_x = unnormalize(candidates.detach(), bounds=standard_bounds)\n",
+    "    new_obj = problem(new_x, dim_y)\n",
+    "    return new_x, new_obj"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Running MOMF optimization \n",
```

Comment:
It seems like you could omit this entire first optimization loop since it is rerun below for the single fidelity qEHVI comparison. I would recommend just running MOMF in one place.

Reply:
We basically wanted to split the tutorial into two parts. One where we could demonstrate how to use MOMF so people who are just interested in getting it to work can just look this first part while the second part is meant to briefly compare its performance with the single fidelity qEHVI. I think this split would make it easier for users to use part of the tutorial for their use case.